protokube/pkg/protokube/volume_mounter.go

```diff
@@ -52,6 +52,12 @@ func (k *VolumeMountController) mountMasterVolumes() ([]*Volume, error) {
 	}
 
 	for _, v := range attached {
+		// Do not ever try to mount root devices
+		if v.LocalDevice == "/dev/sda1" || v.LocalDevice == "/dev/xvda" {
```

Comment:
I think this is just going to bite us later and it shouldn't be needed anyway.  Also presumably the root volumes are always attached?